lab06-kaylee/test/tcp-test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const server = require('../server.js');
+const expect = require('chai').expect;
+const net = require('net');
+//Not sure if I need to require anything else in... TBD.
+
+describe('Server instance', function() {
+  before(done => {
+    server.listen(3000);
+    done();
+  });
+  after(done => {
+    server.close(3000);
+    done();
+  });
+  describe('new client joins chat', function() {
```

Comment:
This test is good, but I won't be issuing bonus points because you're not testing specifically that your ``/nick`` command works.

Also, it's good you scaffolded it out, but the empty ``it`` blocks give me false positives. So just be mindful that your it blocks need ``expect`` statements if you're actually going to test things. So either write full complete tests or don't write them at all (or submit a comment in Canvas that you have false positive tests so we can overlook them).